lib/Utility/OPMLExporter.php

```diff
@@ -90,11 +90,21 @@ public function build(array $folders, array $feeds)
     protected function createFeedOutline(Feed $feed, DOMDocument $document)
     {
         $feedOutline = $document->createElement('outline');
-        $feedOutline->setAttribute('title', $feed->getTitle());
-        $feedOutline->setAttribute('text', $feed->getTitle());
-        $feedOutline->setAttribute('type', 'rss');
-        $feedOutline->setAttribute('xmlUrl', $feed->getUrl());
-        $feedOutline->setAttribute('htmlUrl', $feed->getLink());
+        $attributes = [
+            ['title', $feed->getTitle()],
+            ['text', $feed->getTitle()],
+            ['type', 'rss'],
+            ['xmlUrl', $feed->getUrl()],
+            ['htmlUrl', $feed->getLink()],
+        ];
+
+        foreach ($attributes as $attribute) {
+            if (is_null($attribute[1])) {
+                $feedOutline->setAttribute($attribute[0], "");
+            } else {
+                $feedOutline->setAttribute($attribute[0], $attribute[1]);
+            }
+        }
```

Comment:
I think it is easier to read this way instead of `[0]` being the attibute name and `[1]` the value.
```suggestion
        $attributes = [
            'title' => $feed->getTitle(),
            'text' => $feed->getTitle(),
            'type' => 'rss',
            'xmlUrl' => $feed->getUrl(),
            'htmlUrl' => $feed->getLink(),
        ];

        foreach ($attributes as $name => $value) {
            $feedOutline->setAttribute($name, $value ?? '');
        }
```
If you want, you can also apply the [null coalescing operator (`??`)](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce) in `setAttribute()`, which would simplify the implementation a bit.

Everything else looks fine!